sketch.js

```diff
@@ -1,40 +1,40 @@
-function setup() {
-  createCanvas(600,600);
-
-}
-
-function draw() {
-  background (200,50,0);
+  function setup() {
+    createCanvas(600,600);
   
-  noStroke()
-  fill(300,100,0)
-  rect(0,100,600,100)
+  }
   
-  noStroke()
-  fill(300,200,0)
-  rect(0,200,600,100)
+  function draw() {
+    background (200,50,0);
+    
+    noStroke()
+    fill(300,100,0)
+    rect(0,100,600,100)
```

Comment:
`600` seems to show up a lot, perhaps it would be better set as a variable? You need to declare it outside of the `{``}` for it to be referenced. I'll be explaining them better next class but here's a reference https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var

```javascript
var canvasWidth = 600;
```